composer.json

```diff
@@ -10,7 +10,9 @@
         }
     ],
     "require": {
-        "ezsystems/ezpublish-kernel": "^6.13.4@dev || ^7.3@dev || ^8.0@dev",
+        "ezsystems/ezpublish-kernel": "^8.0@dev",
+        "ezsystems/doctrine-dbal-schema": "^1.0@dev",
```

Comment:
I know that this is added because of:
```
If one of your dependencies has a dependency on an unstable package you need to explicitly require it as well, along with its sufficient stability flag.
```

I've encountered this issue when setting up dependencies for 3.0 - instead of adding the dependency everywhere we went with the approach of adding:
```
    "minimum-stability": "dev",
    "prefer-stable": true,
```
to the packages where it was needed (example: https://github.com/ezsystems/ezplatform-admin-ui/blob/master/composer.json#L15). This change will have to be reverted after we release a stable version of `doctrine-dbal-schema`.

@alongosz @ViniTou maybe we should do the same here, to have an unified approach across all packages?

Reply:
```
    "minimum-stability": "dev",
    "prefer-stable": true,
```
@mnocon hmm, that's interesting. I wasn't a huge fan of setting minimum stability to dev as it made composer install unstable versions of all packages, including 3rd party. So when combined with `prefer-stable` it's not the case?

Reply:
Yes, it installs dev only when there is no stable version available. It works in AdminUI, example build: https://travis-ci.org/ezsystems/ezplatform-admin-ui/jobs/536101037

You can see that it installs correct packages in @dev:
```
  - ...
  - Installing composer/ca-bundle (1.1.4): Loading from cache
  - Installing sensiolabs/security-checker (v5.0.3): Loading from cache
  - Installing sensio/distribution-bundle (v5.0.24): Loading from cache
  - Installing symfony-cmf/routing (2.1.0): Loading from cache
  - Installing ezsystems/ezpublish-kernel (dev-master 03e4495): Loading from cache
  - Installing ezsystems/repository-forms (dev-master 5ed5c97): Loading from cache
  - Installing ezsystems/ezplatform-richtext (dev-master eed18bb): Loading from cache
  - Installing symfony/polyfill-iconv (v1.11.0): Loading from cache
  - Installing egulias/email-validator (2.1.8): Loading from cache
  - Installing swiftmailer/swiftmailer (v6.2.1): Loading from cache
  - Installing symfony/swiftmailer-bundle (v3.2.6): Loading from cache
  - Installing ezsystems/ezplatform-user (dev-master 21d58df): Loading from cache
  - ...
```

Of course it works only because there are no stable versions of our packages for 3.0 - once we release them we will stop getting dev dependencies (that's why we will have to revert that after 3.0 stable is released). But I feel we would do the same with the requirement of `doctrine-dbal-schema` - it's not ideal to have it as a dependency in AdminUI/http-cache/other packages.

Reply:
> Yes, it installs dev only when there is no stable version available. It works in AdminUI, example build: https://travis-ci.org/ezsystems/ezplatform-admin-ui/jobs/536101037

I've also confirmed it just now. This is great workaround, @mnocon another :trophy: :grinning: 

@ViniTou so instead `"ezsystems/doctrine-dbal-schema": "^1.0@dev",` let's add after `require-dev` section: 
```json
    "minimum-stability": "dev",
    "prefer-stable": true,
```